src/test/java/seedu/jarvis/logic/parser/AddCommandParserTest.java

```diff
@@ -79,23 +184,63 @@ public void parseTodo_allFieldsPresent_success() throws ParseException {
+        Event expectedEvent = new Event(VALID_DESCRIPTION, validLocalDateTime);
+        AddTaskCommand newAddTaskCommand = new AddTaskCommand(expectedEvent);
+
+        String userInput = EVENT_PREFIX + " " + VALID_DESCRIPTION + " "
                 + VALID_USERINPUT_DATE + " " + VALID_USERINPUT_TIME;
-        assertParseSuccess(parser, userInput, new AddTaskCommand(expectedEvent));
+        AddCommand expectedAddCommand = parser.parse(userInput);
+        AddTaskCommand typeCastExpectedAddCommand = (AddTaskCommand) expectedAddCommand;
+
+        assertEquals(newAddTaskCommand.getTaskType(), typeCastExpectedAddCommand.getTaskType());
+
+        Event actualTask = (Event) newAddTaskCommand.getTask();
+        Event expectedTask = (Event) typeCastExpectedAddCommand.getTask();
+        assertEquals(expectedEvent, actualTask);
+        assertEquals(actualTask.getDescription(), VALID_DESCRIPTION);
+        assertEquals(actualTask.getLocalDateTime(), validLocalDateTime);
+        assertEquals(actualTask.getDescription(), expectedTask.getDescription());
+        assertEquals(actualTask.getLocalDateTime(), expectedTask.getLocalDateTime());
     }
 
     @Test
-    public void parseDeadline_allFieldsPresent_success() {
-        Deadline expectedDeadline = new Deadline(VALID_DESCRIPTION, VALID_DATE_TIME);
+    public void parseDeadline_allFieldsPresent_success() throws ParseException {
+        DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern(DATE_FORMAT);
+        LocalDateTime validLocalDateTime = LocalDateTime.parse(VALID_DATE_TIME, dateTimeFormat);
+        Deadline expectedDeadline = new Deadline(VALID_DESCRIPTION, validLocalDateTime);
+        AddTaskCommand newAddTaskCommand = new AddTaskCommand(expectedDeadline);
+
         String userInput = DEADLINE_PREFIX + " " + VALID_DESCRIPTION + " "
                 + VALID_USERINPUT_DATE + " " + VALID_USERINPUT_TIME;
-        assertParseSuccess(parser, userInput, new AddTaskCommand(expectedDeadline));
+        AddCommand expectedAddCommand = parser.parse(userInput);
+        AddTaskCommand typeCastExpectedAddCommand = (AddTaskCommand) expectedAddCommand;
+
+        assertEquals(newAddTaskCommand.getTaskType(), typeCastExpectedAddCommand.getTaskType());
+
+        Deadline actualTask = (Deadline) newAddTaskCommand.getTask();
+        Deadline expectedTask = (Deadline) typeCastExpectedAddCommand.getTask();
+        assertEquals(expectedDeadline, actualTask);
+        assertEquals(actualTask.getDescription(), VALID_DESCRIPTION);
+        assertEquals(actualTask.getLocalDateTime(), validLocalDateTime);
+        assertEquals(actualTask.getDescription(), expectedTask.getDescription());
+        assertEquals(actualTask.getLocalDateTime(), expectedTask.getLocalDateTime());
+    }
+
+    @Test
+    public void parseConsultationMasteryCheck_missingInfo_throwsParseException() {
+        assertParseFailure(parser, "-c d/ t/", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
```

Comment:
This is just a suggestion, but perhaps separating these out into static final strings and initializing them at the top might be considered to be better code practice? Since it's easier to update them later and you can reuse them!